index.html

```diff
@@ -42,7 +42,167 @@ <h3 class="social-media-connect">LET’S CONNECT</h3>
+                        <p class="section-p">2015</p>
+                    </div>
+                    <p class="card-body">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="technologies-titles">
+                        <li class="techno">html</li>
+                        <li class="techno">css</li>
+                        <li class="techno">javascript</li>
+                    </ul>
+                    <a class="project-button" href="#">See Project</a>
+                </div>
+            </section>
+
+            <section class="cards card-multi">
+                <img class="portfolio-img" src="images/multi-portfolio.svg" alt="Portfolio snapshoot">
+                <div class="card-element">
+                    <h2 class="card-title">Multi-Post <br> Stories</h2>
+                    <div class="card-section">
+                        <p class="section-p">CANOPY</p>
+                        <img class="dot" src="images/dot.svg" alt="dot">
+                        <p class="section-p">Back End Dev</p>
+                        <img class="dot" src="images/dot.svg" alt="dot">
+                        <p class="section-p">2015</p>
+                    </div>
+                    <p class="card-body">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="technologies-titles">
+                        <li class="techno">html</li>
+                        <li class="techno">css</li>
+                        <li class="techno">javascript</li>
+                    </ul>
+                    <a class="project-button" href="#">See Project</a>
+                </div>
+            </section>
+
+            <section class="cards card-tonic">
+                <img class="portfolio-img" src="images/tonic-portfolio-two.svg" alt="Portfolio snapshoot">
+                <div class="card-element">
+                    <h2 class="card-title">Tonic</h2>
+                    <div class="card-section">
+                        <p class="section-p">CANOPY</p>
+                        <img class="dot" src="images/dot.svg" alt="dot">
+                        <p class="section-p">Back End Dev</p>
+                        <img class="dot" src="images/dot.svg" alt="dot">
+                        <p class="section-p">2015</p>
+                    </div>
+                    <p class="card-body">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="technologies-titles">
+                        <li class="techno">html</li>
+                        <li class="techno">css</li>
+                        <li class="techno">javascript</li>
+                    </ul>
+                    <a class="project-button" href="#">See Project</a>
+                </div>
```

Comment:
- [ ] **Required:** As much as possible, your design should be a clone of the template.

Your Design | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/27709832/149304170-b5fc8e87-d393-447e-937a-80584b6de3f3.png) | ![image](https://user-images.githubusercontent.com/27709832/149304308-78005335-2f42-40ae-95d5-a7d101ee9f7c.png)
![image](https://user-images.githubusercontent.com/27709832/149305581-99690f69-f4ed-47f4-93cc-7484d2dd2a07.png) | ![image](https://user-images.githubusercontent.com/27709832/149305060-74db9163-09c2-4acc-9788-8abb7ac5a32c.png)

Reply:
I added .skill-lang (class) for "Languages" and .card-element for "Cards" I styled it so now it looks identical, thank you.